discovery-provider/src/api/v1/transactions.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from flask_restx import Namespace, Resource, fields
+from src.api.v1.helpers import (
+    abort_not_found,
+    extend_transaction_details,
+    make_full_response,
+    pagination_parser,
+    success_response,
+)
+from src.queries.get_audio_transactions_history import get_audio_transactions_history
+from src.queries.query_helpers import SortDirection, TransactionSortMethod
+from src.utils.auth_middleware import auth_middleware
+
+from .models.transactions import transaction_details
+
+logger = logging.getLogger(__name__)
+
+full_ns = Namespace(
+    "transactions", description="Full transaction history related operations"
+)
+
+transaction_history_response = make_full_response(
+    "transaction_history_response",
+    full_ns,
+    fields.List(fields.Nested(transaction_details)),
+)
+
+
+transaction_history_parser = pagination_parser.copy()
+transaction_history_parser.add_argument(
+    "sort_method",
+    required=False,
+    description="The sort method",
+    type=str,
+    choices=TransactionSortMethod._member_names_,
+    default=TransactionSortMethod.date,
+)
+transaction_history_parser.add_argument(
+    "sort_direction",
+    required=False,
+    description="The sort direction",
+    type=str,
+    choices=SortDirection._member_names_,
+    default=SortDirection.desc,
+)
+
+
+@full_ns.route("")
+class GetTransactionHistory(Resource):
```

Comment:
Hmm so the generated SDK method for this seems slightly wrong and probably unusable due to the headers not being sent. Maybe we just disable generating docs for this (doc=False) and call the endpoint manually for now until we can think of how to make the generator work with it